src/main/java/com/blamejared/crafttweaker/impl/recipes/replacement/DefaultExclusionReplacements.java

```diff
@@ -0,0 +1,34 @@
+package com.blamejared.crafttweaker.impl.recipes.replacement;
+
+import com.blamejared.crafttweaker.CraftTweaker;
+import com.blamejared.crafttweaker.api.managers.IRecipeManager;
+import com.blamejared.crafttweaker.api.recipes.GatherReplacementExclusionEvent;
+import com.blamejared.crafttweaker.impl.recipes.wrappers.WrapperRecipe;
+import net.minecraft.item.crafting.IRecipeType;
+import net.minecraft.item.crafting.SpecialRecipe;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod.EventBusSubscriber(modid = CraftTweaker.MODID, bus = Mod.EventBusSubscriber.Bus.FORGE)
+public final class DefaultExclusionReplacements {
+    @SubscribeEvent
+    public static void onGatherReplacementExclusionEvent(final GatherReplacementExclusionEvent event) {
+    
+        final IRecipeManager manager = event.getTargetedManager();
+        
+        if (manager.getRecipeType() == IRecipeType.CRAFTING) {
+            manager.getAllRecipes()
+                    .stream()
+                    .map(WrapperRecipe::getRecipe)
+                    .filter(it -> it instanceof SpecialRecipe)
```

Comment:
Should this be checking `SpecialRecipe` like it is or do what vanilla does and check `IRecipe#isDynamic`. And should it check all the vanilla recipe types if so, as strictly speaking at least based on what I have seen in the recipe book vanilla allows for other recipe types to have "special" variants as well (even though there likely isn't much of a case there would be any, but if say some mod were to add a "special" furnace recipe then it would be able to be automatically added here.